sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_arg.asciidoc

```diff
@@ -234,9 +251,44 @@ annotated_arg(T, PropertyValueTs... values) ->
 template <typename T, typename PropertiesA, typename PropertiesB>
 annotated_arg(annotated_arg<T, PropertiesA>, PropertiesB>) ->
     annotated_arg<T, /* a type that combines the properties of PropertiesA and PropertiesB */>;
+
+// Overloaded operators for raw type
```

Comment:
Is there any particular reason why you declared these as free functions but the annotated_arg op annotated_arg versions as member functions?  As far as I can tell the versions that start with annotated_arg won't behave differently regardless of where you declare them because the difference between the two versions is whether implicit conversions are considered but implicit conversions aren't considered anyways for function templates.  Is there something I'm missing or is this decision truly arbitrary?

Reply:
There does not seem to be any functional differences in this case but it is generally recommended to use non-member function when possible as they offer looser coupling. Additionally, since we the overloaded operators for when the raw type is on the lhs must be non-member functions, using non-member functions for all overloaded operators of annotated_arg and raw type will be more consistent.